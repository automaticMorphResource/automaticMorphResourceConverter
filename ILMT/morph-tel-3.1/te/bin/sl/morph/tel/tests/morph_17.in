1	xAMwo	unk	
1	1981	unk
1	lo	unk	
1	I	unk	
1	prAMwAnni	unk	
1	"	unk	
1	varald	unk	
1	heVritej	unk	
1	"	unk	
1	prakatiMcAru	unk	
1	.	unk	
