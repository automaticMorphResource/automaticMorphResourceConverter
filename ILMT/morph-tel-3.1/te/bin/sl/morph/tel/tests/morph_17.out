1	xAMwo	unk	<fs af='axi,pn,fn,sg,3,,wo,wo'>
2	1981	unk	<fs af='1981,num,,,,,,'>
3	lo	unk	<fs af='lo,avy,,,,,,'>|<fs af='la,avy,,,,,o,o_avy'>
4	I	unk	<fs af='I,avy,,,,,,'>|<fs af='I,avy,,,,,,'>
5	prAMwAnni	unk	<fs af='prAMwaM,n,,sg,,,nu,nu'>|<fs af='prAMwaM,n,,sg,,,ni,ni'>
6	"	unk	<fs af='&bdquo;,punc,,,,,,'>
7	varald	unk	<fs af='varald,n,,sg,,d,0,0'>|<fs af='varald,n,,sg,,o,ti,ti'>
8	heVritej	unk	<fs af='heVritej,n,,sg,,d,0,0'>|<fs af='heVritej,n,,sg,,o,ti,ti'>
9	"	unk	<fs af='&bdquo;,punc,,,,,,'>
10	prakatiMcAru	unk	<fs af='prakatiMcu,v,fm,pl,3,,A,A'>|<fs af='prakatiMcu,v,fm,pl,3,,A,A'>
11	.	unk	<fs af='&dot;,punc,,,,,,'>
